base/libgit2/tag.jl

```diff
@@ -35,6 +35,7 @@ function name(tag::GitTag)
 end
 
 function target(tag::GitTag)
-    oid_ptr = Ref(ccall((:git_tag_target_id, :libgit2), Ptr{Oid}, (Ptr{Void}, ), tag.ptr))
-    return oid_ptr[]
+    oid_ptr = ccall((:git_tag_target_id, :libgit2), Ptr{Oid}, (Ptr{Void}, ), tag.ptr)
+    oid_ptr == C_NULL && return Oid()
```

Comment:
what would need a try catch? how do you actually use an empty Oid? seems like if that were to ever occur, you'd want to know what made it happen and where it came from, not wait to see some other error (or worse, silently wrong and confusing behavior) down the line from it being empty